### DISCUSSION TITLE
Show Annotations on State timeline & Status history panels

### DISCUSSION THREAD

**Discussion by peterklijn** [grafana/grafana#48679] [Dashboards, Panels & Visualizations]
Hi there,

I was wondering if there are any plans to display [Annotations](https://grafana.com/docs/grafana/latest/dashboards/annotations/) on the [State timeline](https://grafana.com/docs/grafana/latest/visualizations/state-timeline/) and [Status history](https://grafana.com/docs/grafana/latest/visualizations/status-history/) panels?

As seen in the printscreen below, annotations are displayed on the Time Series panel, but don't show in the state timeline & status history panels.

<img width="1903" alt="image" src="https://user-images.githubusercontent.com/2742131/166646886-5e140618-46bd-45c2-afcf-4c9e28ec4474.png">

I'm using Grafana v8.4.2

**Comment by peterklijn** [comment]
Relates to https://github.com/grafana/grafana/discussions/40021

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). This was added here https://github.com/grafana/grafana/pull/47887. Feel free to make an [issue](https://github.com/grafana/grafana/issues) if you cannot find a solution and would like to request the feature.
